proto/message_contents/private_preferences.proto

```diff
@@ -15,13 +15,19 @@ option java_package = "org.xmtp.proto.message.contents";
 // preference store. The only current actions are allow and block.
 // Other actions may be added later
 message PrivatePreferencesAction {
-  // Add the given wallet addresses to the allow list
+  // Grant access
   message Allow {
+    // Add the given wallet addresses to the allow list
     repeated string wallet_addresses = 1;
+    // Add the given group_ids to the allow list
+    repeated string group_ids = 2;
   }
-  // Add the given wallet addresses to the block list
+  // Deny access
   message Block {
+    // Add the given wallet addresses to the block list
     repeated string wallet_addresses = 1;
+    // Add the given group_ids to the block list
+    repeated string group_ids = 2;
```

Comment:
I think we are going to want a separate action for these, rather than a second field in the existing action. Otherwise you could specify both in the same action, which is confusing. 

We can safely rename the type names (although all our SDKs will have to update their import statements)

Reply:
Sure.  I updated to pull out `AllowGroup` and `BlockGroup` into their own message types.
and renamed `Allow` to `AllowDM`  and `Block` to `BlockDM`.  I'll update the SDK side here next.